tests/integration/test_array_creation.py

```diff
@@ -181,25 +215,33 @@ def test_arange_with_dtype(args, dtype):
     assert x.dtype == y.dtype
 
 
-def test_arange_negative():
-    with pytest.raises(ValueError):
-        num.arange(-10)  ###np.arange(-10) returns [] successfully
-    with pytest.raises(ValueError):
-        num.arange(2, -10)  ###np.arange(2, -10) returns [] successfully
-
-    with pytest.raises(OverflowError):
-        num.arange(0, num.inf)
-    with pytest.raises(ValueError):
-        num.arange(0, 1, num.nan)
-
-    with pytest.raises(ZeroDivisionError):
-        num.arange(0, 10, 0)
-    with pytest.raises(ZeroDivisionError):
-        num.arange(0.0, 10.0, 0.0)
-    with pytest.raises(ZeroDivisionError):
-        num.arange(0, 0, 0)
-    with pytest.raises(ZeroDivisionError):
-        num.arange(0.0, 0.0, 0.0)
+class TestArrangeErrors:
+
+    def test_negative_sizes(self):
+        with pytest.raises(ValueError):
+            ###np.arange(-10) returns [] successfully
+            num.arange(-10)
+        with pytest.raises(ValueError):
+            ###np.arange(2, -10) returns [] successfully
+            num.arange(2, -10)
+
+    def test_inf(self):
+        with pytest.raises(OverflowError):
+            num.arange(0, num.inf)
+
+    def test_nan(self):
+        with pytest.raises(ValueError):
+            num.arange(0, 1, num.nan)
+
+    def test_zero_division(self):
+        with pytest.raises(ZeroDivisionError):
+            num.arange(0, 10, 0)
+        with pytest.raises(ZeroDivisionError):
+            num.arange(0.0, 10.0, 0.0)
+        with pytest.raises(ZeroDivisionError):
+            num.arange(0, 0, 0)
+        with pytest.raises(ZeroDivisionError):
+            num.arange(0.0, 0.0, 0.0)
```

Comment:
This seems like a good case to parametrize over tuples passed as `*args` especially since there are more cases that can be tested: nonzero (both positive and negative) start values should be covered explicitly

Reply:
Fixed. Thanks.